modules/installation-osp-accessing-api-floating.adoc

```diff
@@ -61,7 +61,7 @@ api.<cluster_name>.<base_domain>.  IN  A  <API_FIP>
 +
 [NOTE]
 ====
-If you do not control the DNS server, you can add the record to your `/etc/hosts` file. This action makes the API accessible to only you, which is not suitable for production deployment but does allow installation for development and testing.
+If you do not control the DNS server, you can add the record to your `/etc/hosts` file with api and *.apps. The /etc/hosts file does not consider wildcard entries, you need an individual URL list for your applications to add in the /etc/hosts file. This action makes the API accessible to only you, which is not suitable for production deployment but does allow installation for development and testing.
```

Comment:
As *.apps is not supported I think something like this: "If you do not control the DNS server, you can add the record to your `/etc/hosts` file. The /etc/hosts file does not consider wildcard entries, you need a list with the individual URLs for your applications in the /etc/hosts file." can be cleared

Reply:
please add ` to /etc/hosts everywhere, to be consistent.

Reply:
Addressed